File: sanidad-backend/src/main/java/com/farmacia/sanidadbackend/inteligencia/perdidas/dto/ResumenPerdidasDTO.java
package com.farmacia.sanidadbackend.inteligencia.perdidas.dto;

import lombok.Data;
import java.math.BigDecimal;

@Data
public class ResumenPerdidasDTO {
    private BigDecimal totalPerdidasVencimiento;
    private BigDecimal totalInmovilizado;
    private Integer cantidadProductosVencidos;
    private Integer cantidadProductosInmoviles;
    private Integer cantidadInconsistencias;
}

File: SanidadMovil/app/src/main/java/com/sanidad/movil/data/remote/dto/ResumenPerdidasDTO.kt
package com.sanidad.movil.data.remote.dto

data class ResumenPerdidasDTO(
    val totalPerdidasVencimiento: Double,
    val cantidadProductosVencidos: Int,
    val totalInmovilizado: Double,
    val cantidadProductosInmoviles: Int,
    val cantidadInconsistencias: Int
)